analytics-integrations/kahuna/src/main/java/com/segment/analytics/internal/integrations/KahunaEcommerceHelper.java

```diff
@@ -0,0 +1,134 @@
+package com.segment.analytics.internal.integrations;
+
+import android.text.TextUtils;
+
+import com.kahuna.sdk.KahunaAnalytics;
+import com.segment.analytics.internal.model.payloads.TrackPayload;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * Kahuna helps mobile marketers send push notifications and in-app messages.
```

Comment:
We can probably remove these docs, they're the same as the ones on KahunaIntegration.java